src/data/datapack_resources.ts

```diff
@@ -77,6 +77,9 @@ export async function getNamespaceResources(namespace: string, location: string)
     const subDirs = await subDirectories(namespaceFolder);
     for (const type of keys(resourceTypes)) {
         const resourceInfo = resourceTypes[type];
+        if (resourceInfo === undefined) {
+            continue;
```

Comment:
I fixed this in https://github.com/Levertion/mcfunction-langserver/commit/02744b812bb31f69b3000f138945dba5544c0bc4